### ISSUE TITLE
Is it possible to skip namespaces with errors and move on to next ones?

### ISSUE THREAD

**Issue by flashblaze** [AI-Northstar-Tech/vector-io#52]
Upon running the export script, I ran into this error and the script stopped. Is there any way to move on to the next namespace in case an error occurs while exporting?

```
Error: 1 validation error for NamespaceMeta██████████████████████████████████████████████████████████████████████████████████████████| 86/86 [00:02<00:00, 34.05it/s]
metric
  Field required [type=missing, input_value={'namespace': 'dev-clsika...ey96iff1y30/i1.parquet'}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.6/v/missing
Traceback (most recent call last):
  File "/home/neeraj/.local/lib/python3.10/site-packages/vdf_io/export_vdf_cli.py", line 56, in main
    run_export(span)
  File "/home/neeraj/.local/lib/python3.10/site-packages/vdf_io/export_vdf_cli.py", line 123, in run_export
    export_obj = slug_to_export_func[args["vector_database"]](args)
  File "/home/neeraj/.local/lib/python3.10/site-packages/vdf_io/export_vdf/pinecone_export.py", line 118, in export_vdb
    pinecone_export.get_data()
  File "/home/neeraj/.local/lib/python3.10/site-packages/vdf_io/export_vdf/pinecone_export.py", line 443, in get_data
    index_meta = self.get_data_for_index(index_name)
  File "/home/neeraj/.local/lib/python3.10/site-packages/vdf_io/export_vdf/pinecone_export.py", line 540, in get_data_for_index
    namespace_meta = NamespaceMeta(
  File "/home/neeraj/.local/lib/python3.10/site-packages/pydantic/main.py", line 171, in __init__
    self.__pydantic_validator__.validate_python(data, self_instance=self)
pydantic_core._pydantic_core.ValidationError: 1 validation error for NamespaceMeta
metric
  Field required [type=missing, input_value={'namespace': 'dev-clsika...ey96iff1y30/i1.parquet'}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.6/v/missing
Final Step: Fetching vectors: 172it [00:02, 64.60it/s]
```

**Comment by dhruv-anand-aintech** [comment]
Hi @flashblaze, sorry this is a bug with the way I have set NamespaceMeta. Fixing now

**Comment by dhruv-anand-aintech** [comment]
Fix is merged. Use:
```bash
pip install vdf_io==0.1.36
```

or just

```bash
pip install -U vdf_io
```


then 
```bash
export_vdf pinecone
```

**Comment by dhruv-anand-aintech** [comment]
there's a slight bug in the way the VDF_META.json is output. Fixed now in 0.1.37
